packages/lesswrong/components/search/SearchPageTabbed.tsx

```diff
@@ -469,6 +367,27 @@ const SearchPageTabbed = ({classes}:{
 
         <div ref={scrollToRef} />
 
+        <Modal
+          open={true}
+          onClose={() => setModalOpen(false)}
+          aria-labelledby="search-filters-modal"
+          aria-describedby="search-filters-modal"
+          style={{display: modalOpen ? 'flex' : 'none'}}
```

Comment:
Rather than apply inline styles, we use `classNames` to conditionally apply styles via className.  Search the codebase for `classNames(` for examples of how this works - it'd be something like `classNames(classes.filtersModal, { [classes.modalOpen]: modalOpen, [classes.modalClosed]: !modalOpen })` (and you'd need to define `modalOpen` and `modalClosed` in the `styles` for this component with `display: 'flex'` and `display: 'none'` respectively).